integration_tests/test_math.py

```diff
@@ -253,6 +253,31 @@ def test_issue_1242():
     assert abs(math.pi - 3.14159265358979323846) < 1e-10
 
 
+def test_frexp():
+    x:f64 = 6.23
+    mantissa:f64
+    exponent:i16
+    mantissa, exponent = frexp(x)
+    assert abs(mantissa - 0.77875) < eps and exponent == i16(3)
+
+    x = 0.8
+    mantissa, exponent = frexp(x)
+    assert abs(mantissa - 0.8) < eps and exponent == i16(0)
+
+    x = 19.74
+    mantissa, exponent = frexp(x)
+    assert abs(mantissa - 0.616875) < eps and exponent == i16(5)
+
+
+def test_isclose():
+    x:f64 = 2.2130
+    y:f64 = 2.2129
```

Comment:
Please also test for negative value combinations. For example:
- x is positive, y is negative.
- x is negative, y is positive
- x is negative, y is negative.